File: Python/A_NivelIniciante/Exercicio_1160.py
leia = int(input())
for x in range(leia):
    anos = 0
    taxa1 = 0
    taxa2 = 0
    linha = input()
    populacaoA,populacaoB,crescimentoA,crescimentoB = linha.split()
    populacaoA = int(populacaoA)
    populacaoB = int(populacaoB)
    crescimentoA = float(crescimentoA)
    crescimentoB = float(crescimentoB)
    taxa1 = crescimentoA/100
    taxa2 = crescimentoB/100
    while populacaoA<=populacaoB and anos<=100:
        populacaoA = populacaoA + int(populacaoA * taxa1)
        populacaoB = populacaoB + int(populacaoB * taxa2)
        anos = anos + 1
    if anos>100:
        print("Mais de 1 seculo.")
    else:
        print("%d anos." %anos)
    

File: Javascript/A_NivelIniciante/Exercicio_1160.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
for(let i = 1;i<=parseInt(lines[0]);i++){
var valores = lines[i].split(" ")
var CidadeA = parseInt(valores[0])
var CidadeB = parseInt(valores[1])
var taxaA = valores[2]/100
var taxaB = valores[3]/100
var anos = 0
while(CidadeA<=CidadeB && anos<=100){
    CidadeA += parseInt(CidadeA*taxaA)
    CidadeB += parseInt(CidadeB*taxaB)
    anos = anos+1  
}
if(anos>100){
    console.log("Mais de 1 seculo.")
}else{
    console.log(anos + " anos.")
}
}